src/app/portlets/content-types/fields/content-type-fields-properties-form/field-properties/default-value-property/default-value-property.component.spec.ts

```diff
@@ -50,37 +61,29 @@ describe('DefaultValuePropertyComponent', () => {
     });
 
     it('should have a input', () => {
-        comp.group = new FormGroup({
-            name: new FormControl('')
-        });
-        comp.property = {
-            name: 'name',
-            value: 'value',
-            field: {}
-        };
-
         fixture.detectChanges();
-
         const pInput: DebugElement = fixture.debugElement.query(By.css('input[type="text"]'));
 
         expect(pInput).not.toBeNull();
     });
 
     it('should have a field-message', () => {
-        comp.group = new FormGroup({
-            name: new FormControl('')
-        });
-        comp.property = {
-            name: 'name',
-            value: 'value',
-            field: {}
-        };
-
         fixture.detectChanges();
-
         const fieldValidationmessage: DebugElement = fixture.debugElement.query(By.css('dot-field-validation-message'));
 
         expect(fieldValidationmessage).not.toBeNull();
         expect(comp.group.controls['name']).toBe(fieldValidationmessage.componentInstance.field);
     });
+
+    it('set error label to the default value', () => {
+        fixture.detectChanges();
+
+        expect(comp.errorLabel).toEqual('default error');
+    });
+    it('set error label to specific valid class mapped', () => {
+        comp.property.field.clazz = 'com.dotcms.contenttype.model.field.ImmutableDateField';
+        fixture.detectChanges();
+
+        expect(comp.errorLabel).toEqual('date error');
+    });
```

Comment:
we missed a test for 'com.dotcms.contenttype.model.field.ImmutableDateTimeField''